internal/storage/v2/clickhouse/tracestore/reader.go

```diff
@@ -48,3 +50,27 @@ func (r *Reader) GetServices(ctx context.Context) ([]string, error) {
 	}
 	return services, nil
 }
+
+func (r *Reader) GetOperations(
+	ctx context.Context,
+	query tracestore.OperationQueryParams,
+) ([]tracestore.Operation, error) {
+	rows, err := r.conn.Query(ctx, sqlSelectOperations, query.ServiceName, query.SpanKind)
```

Comment:
do we need special handling for the case where the query has an empty SpanKind?

Reply:
yes it means load all kinds